examples/pytorch/rgcn/README.md

```diff
@@ -51,3 +43,14 @@ FB15k-237 in Filtered-MRR
 ```
 python link.py --gpu 0 --eval-protocol filtered
 ```
+
+Summary 
+-------
+### Entity Classification
```

Comment:
1. How good/bad are these results? Do you have numbers from the paper for reference?
2. Why do you only include the results for `entity.py`?

Reply:
For (1), the results are consistent with the older version. Some of the results are not as good compared to ref.  The numbers (performances) from the paper are tuned separately for each case. I will see if I can reproduce using their settings for each individual case.

For (2), I haven't tested/looked at link pred results yet. I will update this section by then. For entity classification results using sampling method (`entity_sample.py`), I can only observe negligible differences compared to full-graph approach.